### ISSUE TITLE
[Doc] proxy authentication configuration 

### ISSUE THREAD

**Issue by RaigorJiang** [apache/shardingsphere#10469]
Due to the authority API changed, the proxy documentation needs to be updated.

### Currently 
```
users:
   - root@:root # <username>@<hostname>:<password>
   - sharding@%:sharding
```
### New
```
rules:
  - !AUTHORITY
    users:
      - root@%:root
      - sharding@:sharding
    provider:
      type: NATIVE
```

### Files
- authentication.cn.md 
- authentication.en.md

refer #10251

**Comment by menghaoranss** [comment]
@RaigorJiang Could I assign this issue to you?

**Comment by RaigorJiang** [comment]
@RaigorJiang  OK, please assign to me, thank you.

**Comment by RaigorJiang** [comment]
Closed in #10556
